docs/user-guide/cli-using-global-profile-configuration.md

```diff
@@ -0,0 +1,535 @@
+
+The user name and password fields are not supplied in the service definitions.
+
+**Example:** <!--Not sure what this is trying to same, as there seems to be something missing --> The username and password fields for ZOSMF1 and ZOSMF2 are not supplied in the service definitions to allow them to inherit values from the base array:
+
+```json
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "lpar1": {
+            "properties": {
+                "host": "example1.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 443
+                    }
+                }
+            }
+        },
+        "lpar2": {
+            "properties": {
+                "host": "example2.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 1443
+                    }
+                }
+            }
+        },
+        "my_base": {
+            "type": "base",
+            "properties": {
+                "rejectUnauthorized": true
+            },
+            "secure": [
+                "user",
+                "password"
+            ]
+        }
+    },
+    "defaults": {
+        "zosmf": "lpar1.zosmf",
+        "base": "my_base"
+    },
+    "plugins": []
+}
+```
+
+### Tips for using the base array
+
+Use the base array to share option values between services. You might share option values between services in the following situations:
+- You have multiple services that share the same username, password, or other value.
+- You want to store a web token to access all services through Zowe API Mediation Layer.
+- You want to trust a known self-signed certificate or your site does not have server certificates configured. You can define `reject-unauthorized` in the base array with a value of  `false` to apply to all services. Understand the security implications of accepting self-signed certificates at your site before you use this method.
+
+## Sharing global configuration
+
+You might want to share a configuration globally:
+- With developers so that they can begin working with a defined set of mainframe services. The recipient of the file manually places it in their local `~/.zowe` folder before issuing CLI commands.
+- To add to your project directory in an SCM tool such as GitHub. This lets other developers pull the project to their local machine and make use of the defined configuration. Zowe CLI commands that you issue from within the project directory automatically use the project's config scheme.
+- To enable test automation and CI/CD, letting your pipelines make use of the project configuration.
+
+## Example configurations
+
+**Example 1:** The settings are accessing multiple services directly on multiple LPARs that share the same username and password.
+
+```json
+{
```

Comment:
Similar issues as previous examples.

Reply:
Updated the description for this example to mention that it uses nested profiles. They are still valid for this use case, just not enabled by default.

An example of "accessing multiple services directly on multiple LPARs that share the same username and password" without using nested profiles is also provided earlier in this document.